pkg/alerts/nrql_conditions.go

```diff
@@ -731,6 +732,7 @@ const (
       closeViolationsOnExpiration
       expirationDuration
       openViolationOnExpiration
+			infraHostCleanShutdown
     }
```

Comment:
:memo: I have no idea why the indentation appears so off here. Locally, everything looks fine